ops/ccs-ops-misc/load_test/common/bene_tests.py

```diff
@@ -1,29 +1,40 @@
 '''Locust tests that require a pool of beneficiary IDs.'''
 
 import random
-
+from typing import List
+from locust import events
+from locust.env import Environment
 from common.bfd_user_base import BFDUserBase
+from common.locust_utils import is_distributed, is_locust_master
 from common.url_path import create_url_path
 from common import data, db
 
+table_sample_bene_ids = True
+master_bene_ids: List[str] = []
+
+@events.init.add_listener
+def _(environment: Environment, **kwargs):
```

Comment:
```suggestion
def _(environment: Environment):
```

Reply:
The unused `**kwargs` here is actually a pattern suggested by Locust's [documentation on event hooks](https://docs.locust.io/en/stable/extending-locust.html#event-hooks). Essentially, they're suggesting to use `**kwargs` to guard against unexpected breakage in the case that an existing event hook gets refactored to have another argument. 

I've actually run into this already with one of their event hooks where their documentation was inaccurate and omitted an additional argument that it had (but was unnecessary for my usage). Using `**kwargs` guards against that here.